src/main/resources/com/rultor/agents/req/merge.sh

```diff
@@ -20,6 +20,12 @@ while [ $(git show-branch "${BRANCH}" 2>/dev/null | wc -l) -gt 0 ]; do
 done
 
 git checkout -B "${BRANCH}" "fork/${fork_branch}"
+git fetch origin
+if git log --name-status --oneline HEAD..origin/${head_branch} | grep ".rultor.yml"; then
+  echo "Since original repository has different version of .rultor.yml file you have to sync your fork with the upstream repository."
+  exit 1
+fi
```

Comment:
@mkordas

>  Will it prohibit any changes to .rultor.yml file?

No. You can change `.rultor.yml` as you wish. But when you try to merge, my script will check that your merge is synced with upstream. This check occurs only when `.rultor.yml` was changed in master repository. We need this to be sure that project will not crash after merge (see description #1039 for details).